.cargo/config.toml

```diff
@@ -1,3 +1,8 @@
+[build]
+rustflags = [
+  "-Wclippy::assertions_on_result_states",
+]
```

Comment:
This should be a `#![warn(clippy::assertions_on_result_states)]` in `lib.rs` :)

We did it like this in Firecracker for unrelated reasons that don't apply to rust-vmm

Reply:
> This should be a `#![warn(clippy::assertions_on_result_states)]` in `lib.rs` :)
> 
> We did it like this in Firecracker for unrelated reasons that don't apply to rust-vmm

Noted